kitchen.yml

```diff
@@ -57,14 +62,26 @@ verifier:
     - path: test/integration/default
 
 suites:
-  - name: deb
+  - name: debian
     excludes:
       - centos-7
       - fedora-27
+      - ubuntu-16.04
+      - ubuntu-18.04
+
+  - name: ubuntu
+    excludes:
+      - debian-9
+      - fedora-27
+      - centos-7
+    provisioner:
+      pillars-from-files:
+        packages.sls: test/integration/default/pillar.example.ubuntu
```

Comment:
This is causing the tests to fail for `16.04` and `18.04`:

```travis
Failed to complete #converge action: [No such file or directory @ rb_sysopen - test/integration/default/pillar.example.ubuntu] on ubuntu-ubuntu-1604
```

Reply:
Typo in the naming. Fixed.